code/modules/mob/living/silicon/robot/emote.dm

```diff
@@ -237,6 +237,15 @@
 			else
 				src << "You are not security."
 
+		if("bark")
+			if (istype(module,/obj/item/weapon/robot_module/robot/knine) || istype(module,/obj/item/weapon/robot_module/robot/medihound) || istype(module,/obj/item/weapon/robot_module/robot/scrubpup) || istype(module,/obj/item/weapon/robot_module/robot/ert) || istype(module,/obj/item/weapon/robot_module/robot/science) || istype(module,/obj/item/weapon/robot_module/robot/engiedog) || istype(module,/obj/item/weapon/robot_module/robot/clerical/brodog))
```

Comment:
I feel like this could probably be made into a var check like main's dogborgs have for icon offsets.